cslage/lab_notebooks/test_notebook_20210107.ipynb

```diff
@@ -0,0 +1,859 @@
+{
```

Comment:
<p>For personal notebooks (in your directory), please make sure that they are all cleared of contents.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-ts/ts_notebooks/pull/42/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>